lib/tasks/test_schemas.rake

```diff
@@ -0,0 +1,11 @@
+require "expressir"
```

Comment:
We can just use the command line here to do it without a new Rake task:
(see https://github.com/metanorma/iso-10303/blob/main/.github/workflows/validate_express.yml)
```sh
bundle exec expressir validate schemas-srl.yml
bundle exec expressir validate schemas_module.yml
bundle exec expressir validate schemas_bom.yml
```

Reply:
This is also because not all `*.exp` files in the iso-10303 repository are valid (some are not meant to be tested).